openmdao/core/group.py

```diff
@@ -959,6 +959,30 @@ def clear_dparams(self):
             if isinstance(system, Group):
                 system.clear_dparams()  # only call on Groups
 
+    def pre_setup(self, problem):
+        """
```

Comment:
Iis this something we might want to make available for all systems, and not just Groups?